### PR TITLE
zipalign: add page

### PR DESCRIPTION
**Pull request by Rudloff**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/zipalign.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# zipalign
+
+> Zip archive alignment tool.
+> More information: <https://developer.android.com/studio/command-line/zipalign>.
+
+- Align the data of a ZIP file on {{number}} bytes:
```
Comment (agnivade) [inline pages/common/zipalign.md]: Let's use a concrete value for number

File: pages/common/zipalign.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`zipalign {{number}} {{path/to/input.zip}} {{path/to/output.zip}}`
+
+- Check that a ZIP file is correctly aligned on {{number}} bytes:
```
Comment (agnivade) [inline pages/common/zipalign.md]: Same as above.

File: pages/common/zipalign.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+`zipalign -v -c {{number}} {{path/to/input.zip}}`
```
Comment (agnivade) [inline pages/common/zipalign.md]: `-v` is for verbose right ? We should also mention that in the description.

Comment (Rudloff) [reply pages/common/zipalign.md]: Indeed. (Because zipalign is quiet by default and without it, it would just return an exit code.)

Comment (agnivade) [reply pages/common/zipalign.md]: You have not mentioned the verbosity part in the description.

Comment (Rudloff) [reply pages/common/zipalign.md]: Fair enough, I added your suggestion.

File: pages/common/zipalign.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# zipalign
+
+> Zip archive alignment tool.
+> More information: <https://developer.android.com/studio/command-line/zipalign>.
+
+- Align the data of a ZIP file on {{4}} bytes:
```
Comment (mebeim) [inline pages/common/zipalign.md]:
```suggestion
- Align the data of a ZIP file on 4-byte boundaries:
```

File: pages/common/zipalign.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+`zipalign {{4}} {{path/to/input.zip}} {{path/to/output.zip}}`
+
+- Check that a ZIP file is correctly aligned on {{4}} bytes and display the results in a verbose manner:
```
Comment (mebeim) [inline pages/common/zipalign.md]:
```suggestion
- Check that a ZIP file is correctly aligned on 4-byte boundaries and display the results in a verbose manner:
```

File: pages/common/zipalign.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# zipalign
+
+> Zip archive alignment tool.
```
Comment (mebeim) [inline pages/common/zipalign.md]:
```suggestion
> Zip archive alignment tool.
> Part of the Android SDK build tools.
```
Let's mention that this command is part of the Android SDK.

**Pull request merged**